src/state/action-creators.js

```diff
@@ -450,7 +452,7 @@ export const lift = (id: DraggableId,
   setTimeout(() => {
     const state: State = getState();
 
-    if (state.phase !== 'IDLE' || state.phase !== 'DRAG_COMPLETE') {
+    if (state.phase !== 'IDLE' && state.phase !== 'DRAG_COMPLETE') {
```

Comment:
Can you explain the thinking around this one a little more?

Reply:
I think it makes sense, but good to walk through

Reply:
Was this line the bug?

Reply:
nah this was just something I noticed while I was debugging. `state.phase !== 'IDLE' || state.phase !== 'DRAG_COMPLETE'` will always evaluate to `true`.